components/script/dom/webidls/HTMLElement.webidl

```diff
@@ -17,10 +17,16 @@ interface HTMLElement : Element {
   readonly attribute DOMStringMap dataset;
 
   // microdata
-  //         attribute boolean itemScope;
+  //           attribute boolean itemScope;
+  //           attribute DOMString itemProp;
+
   //         attribute DOMString itemId;
   //readonly attribute HTMLPropertiesCollection properties;
   //         attribute any itemValue; // acts as DOMString on setting
+  [Pref="dom.microdata.enabled"]
```

Comment:
Since these are only for use in test code, let's use the dom.microdata.testing.enabled name instead.